tests/command.dev.test.js

```diff
@@ -6,118 +6,104 @@ const FormData = require('form-data')
 const { withSiteBuilder } = require('./utils/siteBuilder')
 const { startExternalServer } = require('./utils/externalServer')
 
-test('should return index file when / is accessed', async t => {
-  await withSiteBuilder('site-with-index-file', async builder => {
-    builder.withContentFile({
-      path: 'index.html',
-      content: '<h1>⊂◉‿◉つ</h1>',
-    })
+const testMatrix = [
```

Comment:
The changes in this file is for support a test matrix with different args (and creating unique test names)